src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -330,16 +353,23 @@ public SNIHandle CreateConnectionHandle(object callbackObject, string fullServer
             return sniHandle;
         }
 
-        private static byte[] MakeMsSqlServerSPN(string fullyQualifiedDomainName, int port = DefaultSqlServerPort)
+        private static byte[] GetSqlServerSPN(string hostNameOrAddress, string instanceName = null)
```

Comment:
`string instanceName = null` 
This doesn't seem like an optional parameter based on the calls to GetSqlServerSPN.
This should be `private static byte[] GetSqlServerSPN(string hostNameOrAddress, string instanceName)`